src/core/rawlog.c

```diff
@@ -130,6 +130,7 @@ void rawlog_open(RAWLOG_REC *rawlog, const char *fname)
 			      log_file_create_mode);
 	if (rawlog->handle == -1) {
 		g_warning("rawlog open() failed: %s", strerror(errno));
+		return;
```

Comment:
You're right, thanks!  (That's what happens when you try to fix things before the morning coffee...)